src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/CSharpForwardCancellationTokenToInvocations.Fixer.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Operations;
+using Microsoft.NetCore.Analyzers.Runtime;
+
+namespace Microsoft.NetCore.CSharp.Analyzers.Runtime
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp)]
+    public sealed class CSharpForwardCancellationTokenToInvocationsFixer : ForwardCancellationTokenToInvocationsFixer
+    {
+        protected override bool TryGetInvocation(
+            SemanticModel model,
+            SyntaxNode node,
+            CancellationToken ct,
+            [NotNullWhen(true)] out IInvocationOperation? invocation)
+        {
+            invocation = null;
+
+            IOperation operation;
+
+            // If the method was invoked using nullability for the case of attempting to dereference a possibly null reference,
+            // then the node.Parent.Parent is the actual invocation (and it will contain the dot as well)
+            if (node.Parent is MemberBindingExpressionSyntax)
+            {
+                operation = model.GetOperation(node.Parent.Parent, ct);
+            }
+            else
+            {
+                operation = model.GetOperation(node.Parent, ct);
+            }
+
+            if (operation is IInvocationOperation invocationOperation)
+            {
+                invocation = invocationOperation;
+            }
```

Comment:
```suggestion
            invocation = operation as IInvocationOperation;
```

Note that you can then get rid of line 24.